README.md

```diff
@@ -23,6 +23,25 @@ This will generate trait implementations for `Example` for any `T`,
 as opposed to std's derives, which would only implement these traits with
 `T: Trait` bound to the corresponding trait.
 
+Multiple `derive_where` attributes can be added to an item, but only the
+first one should use any path qualifications. Otherwise helper attributes
+won't be applied to any but the first `derive_where` attribute on the item.
+
+```rust
+#[derive_where::derive_where(Clone, Debug)]
+#[derive_where(Default, Hash)]
+struct Example<T>(PhantomData<T>);
+```
+
+Multiple `derive_where` attributes can be added to an item, but only the
+first one must use any path qualifications.
```

Comment:
That sounds a bit repetitive, also half is missing in the docs

Reply:
This is wrong ... fixing it now.

Reply:
Oh, I don't know where you got this from, but this is outdated?

Reply:
No it is not ... I don't know whats happening ...

Reply:
All right, I didn't see that it was duplicate ... it's good now, thanks!